pkg/kind/kind.go

```diff
@@ -38,6 +45,50 @@ func SetUp() error {
 }
 
 func createKindCluster() error {
-	fmt.Println("TODO: Creating Kind cluster...")
+	fmt.Println("Creating Kind cluster...")
+
+	// Get kind config file
+	kindConfig, err := ioutil.TempFile(os.TempDir(), "kind-config-*.yaml")
+	if err != nil {
+		return fmt.Errorf("kind create: %w", err)
+	}
+
+	defer os.Remove(kindConfig.Name())
+
+	configRaw := "kind: Cluster\n" +
```

Comment:
You can use ` for multi line strings in go

Reply:
for some reason, this was more aesthetically pleasing to my eye (which admittedly is not necessarily the best criteria to use)